soroban-env-host/src/host.rs

```diff
@@ -185,6 +194,10 @@ impl Host {
         }))
     }
 
+    pub fn set_source_account(&self, source_account: Option<AccountId>) {
```

Comment:
Why does this take an `Option`?

Reply:
To unset the source_account?

Reply:
To unset it since the value starts out at None. I see looking at ledger info we don't allow unsetting it though. I'll remove the Option on this line.